openupgradelib/openupgrade.py

```diff
@@ -564,6 +564,9 @@ def update_module_names(cr, namespec):
         query = ("UPDATE ir_module_module_dependency SET name = %s "
                  "WHERE name = %s")
         logged_query(cr, query, (new_name, old_name))
+        query = ("UPDATE ir_translation SET module = %s "
```

Comment:
Is it an update or a delete? Hypothesis for delete: Module data is being loaded  by actualizing the module.